packages/stylelint-config-terra/README.md

```diff
@@ -0,0 +1,85 @@
+
+## What is Stylelint?
+
+[Stylelint](https://stylelint.io/) is a mighty, modern CSS linter and fixer that helps you avoid errors and enforce consistent conventions in your stylesheets.
+
+## Installation
+
+Install the module
+
+```shell
+$ npm install stylelint --save-dev
+$ npm install @cerner/stylelint-config-terra --save-dev
+```
+
+## Usage
+
+### package.json
+
+```json
+{
+  "stylelint": {
+    "extends": "@cerner/stylelint-config-terra"
+  }
+}
+```
+
+### Extending Terra's Configuration
+
+It is possible to specify and override the rules defined by stylelint-config-terra. Read more about it [here](https://stylelint.io/user-guide/configuration/#extends).
+
+For example, it is possible to override the browsers specified to the no-unsupported-browser-features plugin.
+
+```json
+{
+  "stylelint": {
+    "extends @cerner/stylelint-config-terra",
+    "rules": {
+      "plugin/no-unsupported-browser-features": [
+        true,
+        "browsers": ["iOS >= 10"],
+        "severity": "warning",
+      ]
+    }
+  }
+}
+```
+
+## Custom Lint Rules
+
+The following custom rules are enabled by default.
+
+* [terra/custom-property-name](https://github.com/cerner/terra-toolkit/blob/main/packages/stylelint-config-terra/src/rules/custom-property-name): Requires custom properties to be suffixed with the css property name.
+* [terra/custom-property-namespace](https://github.com/cerner/terra-toolkit/blob/main/packages/stylelint-config-terra/src/rules/custom-property-namespace): Requires custom properties to be prefixed with a namespace.
+* [terra/custom-property-pattern](https://github.com/cerner/terra-toolkit/blob/main/packages/stylelint-config-terra/src/rules/custom-property-pattern): Requires custom properties to be written in lowercase alphanumeric characters and hyphens.
+* [terra/custom-property-pseudo-selectors](https://github.com/cerner/terra-toolkit/blob/main/packages/stylelint-config-terra/src/rules/custom-property-pseudo-selectors): Requires custom properties to include all ancestor pseudo selectors in order.
+
+## Versioning
+
+stylelint-config-terra is considered to be stable and will follow [SemVer](http://semver.org/) for versioning.
+
+1. MAJOR versions represent breaking changes
+2. MINOR versions represent added functionality in a backwards-compatible manner
+3. PATCH versions represent backwards-compatible bug fixes
+
+Consult the component CHANGELOGs, related issues, and PRs for more information.
```

Comment:
The versioning and Contribution sections can be removed not hat it is in the mono-repo right?